File: lua/refactoring/tests/refactor/119/php/example/extract_var.expected.php
<?php

class Order 
{
    public $quantity;
    public $itemPrice;
}

class testing 
{
    public function orderCalculation(
        Order $order, int $blah, int $nonsence
    ) 
    {
        $blah * $nonsence;
        $basePrice = $order->quantity 
            *
            $order->itemPrice;
        $basePrice;

        return $basePrice - 
            max(0, $order->quantity - 500) * $order->itemPrice * 0.5 +
            min($basePrice * 0.1, 100);
    }
}

File: lua/refactoring/tests/refactor/119/rb/example/extract_var.expected.rb
def orderCalculation(order, a, b)
  temp = a * b
  basePrice = order.quantity * order.itemPrice
  blah = (basePrice) - 7

  puts(temp, blah)

  basePrice - max(0, order.quantity - 500) * order.itemPrice * 0.05 + min(basePrice * 0.01, 100)
end
